webui/react/src/e2e/tests/workspaceTasks.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { expect, test } from 'e2e/fixtures/global-fixtures';
+import { TaskLogs } from 'e2e/models/pages/TaskLogs';
+import { WorkspaceDetails } from 'e2e/models/pages/WorkspaceDetails';
+
+test.describe('Workspace Tasks', () => {
+  test('JupyterLab', async ({ authedPage, newWorkspace, context }) => {
+    const workspaceDetails = new WorkspaceDetails(authedPage);
+
+    await workspaceDetails.gotoWorkspace(newWorkspace.response.workspace.id);
+    await workspaceDetails.tasksTab.pwLocator.click();
+
+    await workspaceDetails.taskList.jupyterLabButton.pwLocator.click();
+    await workspaceDetails.taskList.jupyterLabModal.pwLocator.waitFor();
+    await workspaceDetails.taskList.jupyterLabModal.footer.submit.pwLocator.click();
+    await workspaceDetails.taskList.jupyterLabModal.pwLocator.waitFor({ state: 'hidden' });
```

Comment:
could you split the test into a few steps? I think the modal would be a good place to split steps